### ISSUE TITLE
Add Unit Examples?

### ISSUE THREAD

**Issue by ericdmoore** [mtrunkat/js-flowage#1]
i'm trying to compare this package to `mississippi`

Do you think it makes sense to add some unit examples to your code base of what you think a normal usage pattern is - so that I can make the case of if this make sense for my use case - or keep using the `ol'mississippi`

**Ref**
1: [npm.im/mississippi](https://npm.im/mississippi)

**Comment by mtrunkat** [comment]
Definitely. I am just about to add more examples once I found some free time.

The main difference I see is that `mississipi` is a general stream utility collection where `flowage` is only for object streams. This is a limitation but on the other hand, it allows more features to be added. I have just updated description and README with "object stream" keywords to make this clear.

**Comment by ericdmoore** [comment]
Nice!
Once I have my head wrapped around your project, and project goals. I'd be happy to help out when you are ready - and have an idea of what you want help with
